lib/otp/views/configure_static_dialog.dart

```diff
@@ -29,6 +29,7 @@ import '../../widgets/app_text_field.dart';
 import '../../widgets/choice_filter_chip.dart';
 import '../../widgets/responsive_dialog.dart';
 import '../keys.dart' as keys;
+import '../keys.dart';
```

Comment:
remove this and use the prefix (instead of generateSecretKey use keys.generateSecretKey and so on)